ercc/Makefile

```diff
@@ -1,6 +1,28 @@
-.PHONY: all
+# Copyright 2019 Intel Corporation
```

Comment:
If you want to add this here, please also add IBM copyright as well as this file I have created already.

Reply:
Marcus, I cannot for obvious reasons. But I am happy to wait for you to edit this file
* either directly in this PR and add a commit (you can do it throught the web interface, don't forget the signoff)
* or in another PR to be merged before or after this one.

Reply:
ok, I created #72 which adds missing licenses.

Reply:
so let's merge this first and then rebase.

Reply:
sounds good